<launch>

  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find lwr_ros_control)/config/lwr_control.yaml" command="load"/>

  <!-- load position controllers -->
<!--   <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
    output="screen"  args="joint_state_controller joint0_position_controller joint1_position_controller joint2_position_controller joint3_position_controller joint4_position_controller joint5_position_controller joint6_position_controller"/> -->

  <!-- load trajectory controllers. This is used in the moveit demo -->
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
    output="screen"  args="joint_state_controller lwr_trajectory_controller"/>

  <!-- convert joint states to TF transforms for rviz, etc -->
<!--   <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
    respawn="false" output="screen"> -->
    <!-- <remap from="/joint_states" to="/lwr/joint_states" /> -->
  <!-- </node> -->

</launch>